WebContent/certified.html

```diff
@@ -58,5 +58,9 @@ <h1 class="translate" data-i18n="Conforming Organizations">Conforming Organizati
     
   
 </body>
-
+<script src="resources/lib/url/url.min.js"></script>  
+<script src="resources/i18next.min.js" ></script>
+<script src="resources/jquery-i18next.min.js" ></script>
+<script src="resources/i18nextXHRBackend.js"></script>
+<script src="resources/global.js"></script>
 </html>
```

Comment:
Please move the script references to the top of the HTML page with the rest of the script references (code readibility / maintainability issue)